tests/robottelo/test_cli.py

```diff
@@ -1,7 +1,8 @@
 import six
 import unittest2
 
-from robottelo.cli.base import Base, CLIReturnCodeError, CLIError
+from robottelo.cli.base import (Base, CLIReturnCodeError, CLIError,
+                                CLIBaseError, CLIDataBaseError)
```

Comment:
I prefer to use the style:

```python
from robottelo.cli.base import (
    Base,
    CLIError,
    ...
)
```

It is just a matter of style, but also helps when we need to add/remove imports and avoid long line indentation.
```